definitions/divorce/json/CaseEventToFields.json

```diff
@@ -7083,7 +7075,7 @@
     "LiveFrom": "16/04/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorSOTDN",
-    "CaseFieldID": "SeparationTimeLivedTogetherDetailsDN\t",
```

Comment:
WTH did this ever work

Reply:
lol yeah I think it's only used on the front-end in these instances, other references used elsewhere don't have the tab